packages/remix-cloudflare-workers/package.json

```
@@ -18,9 +18,9 @@
     "@remix-run/cloudflare": "1.3.3"
   },
   "peerDependencies": {
-    "@cloudflare/workers-types": "^2.2.2"
+    "@cloudflare/workers-types": "^3.2.0"
```

This would be a breaking change & is already handled by #2530

```suggestion
    "@cloudflare/workers-types": "^2.2.2"
```